### ISSUE TITLE
fyi - 0.17.0-alpha.11 - swift generation failing / ts working

### ISSUE THREAD

**Issue by johndpope** [apollographql/apollo-codegen#260]
https://github.com/apollographql/apollo-codegen/commit/337b6b76d313d1fbf5515958552d5cb0585ca8de

on this specific branch when I run
 apollo-codegen generate corewatchlist.graphql --schema schema.json --output schema.swift

files here
https://gist.github.com/johndpope/7cc331ad7182fb5ed112738500af5383

swift fails
```
apollo-codegen generate *.graphql --schema schema.json --output schema.swift
Error: ENOENT: no such file or directory, stat '/Users/johndpope/Documents/gitWorkspace/apollo-codegen/schema.swift'
    at Object.fs.statSync (fs.js:954:11)
    at Object.generateSource (/usr/local/lib/node_modules/apollo-codegen/src/swift/codeGeneration.ts:44:10)
    at Object.generate (/usr/local/lib/node_modules/apollo-codegen/src/generate.ts:35:28)
    at Object.yargs.command.command.command.argv [as handler] (/usr/local/lib/node_modules/apollo-codegen/src/cli.js:188:7)
    at Object.self.runCommand (/usr/local/lib/node_modules/apollo-codegen/node_modules/yargs/lib/command.js:233:22)
    at Object.Yargs.self._parseArgs (/usr/local/lib/node_modules/apollo-codegen/node_modules/yargs/yargs.js:990:30)
    at Object.get [as argv] (/usr/local/lib/node_modules/apollo-codegen/node_modules/yargs/yargs.js:927:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/apollo-codegen/lib/cli.js:178:5)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
```

however - 
typescript/ scala  all seem working fine with target --target ts
apollo-codegen generate *.graphql --schema schema.json --target ts --output schema.ts

perhaps we need --target swift ???

apollo-codegen generate *.graphql --schema schema.json --target swift --output schema.swift also failing.

**Comment by johndpope** [comment]
fyi - 
it appears there's some updated dependencies available
npm install -g npm-check
<img width="362" alt="screen shot 2017-10-03 at 11 50 48 am" src="https://user-images.githubusercontent.com/289994/31134972-aff6474a-a831-11e7-97ac-0e04a3e39114.png">

**Comment by zfoltin** [comment]
It seems that the `--output` param has to be a directory instead of a Swift file. The tool now creates a Swift file for each .graphql query.

**Comment by martijnwalraven** [comment]
Sorry, it turns out adding experimental support for outputting multiple files in 0.17.0-alpha.11 broke outputting a single file. This should be fixed in 0.17.0-alpha.11.

But if you want to output multiple files, you can indeed specify a directory for `--output`.
